frontend/device/generic/device.lua

```diff
@@ -199,7 +199,7 @@ function Device:retrieveNetworkInfo()
     if std_out then
         local result = std_out:read("*all")
         std_out:close()
-        std_out = io.popen('iwconfig eth0 | grep ESSID | cut -d\\" -f2')
+        std_out = io.popen('iwconfig | grep ESSID | cut -d\\" -f2')
```

Comment:
Btw, what's the device on the Touch N905? In any case as written this won't work right as a generic statement.

As written:
```sh
# iwconfig | grep ESSID
lo        no wireless extensions.

usb0      no wireless extensions.

eth0      IEEE 802.11-DS  ESSID:"Pink_P"  Nickname:""
```
In other words, you'd get this whole thing:
```sh
# iwconfig | grep ESSID | cut -d\" -f2
lo        no wireless extensions.

usb0      no wireless extensions.

Pink_P
```

Before:
```sh
# iwconfig eth0 | grep ESSID
eth0      IEEE 802.11-DS  ESSID:"Pink_P"  Nickname:""
```

Redirecting stderr to /dev/null works out for me:
```sh
# iwconfig 2>/dev/null | grep ESSID
eth0      IEEE 802.11-DS  ESSID:"Pink_P"  Nickname:""
```